PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+*Note*: Before submitting this pull request, please review our [contributing guidelines](https://github.com/tomchristie/django-rest-framework/blob/master/CONTRIBUTING.md#pull-requests).
+
+## Description
+Please describe your pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```

Comment:
"When linking to an issue, please use `refs #...` in the description of the pull request."